base/applications/mspaint/palette.cpp

```diff
@@ -86,7 +126,7 @@ LRESULT CPaletteWindow::OnLButtonDblClk(UINT nMsg, WPARAM wParam, LPARAM lParam,
 
 LRESULT CPaletteWindow::OnRButtonDblClk(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
 {
-    if (GET_X_LPARAM(lParam) >= 31)
+    if (GET_X_LPARAM(lParam) && !bk)
```

Comment:
```suggestion
    if (GET_X_LPARAM(lParam) >= 31 && !bk)
```

You're also might want to check the return value of `GET_X_LPARAM` here. (wrong copypasta?)

Reply:
Yes. Thank you 👍